anybox/recipe/odoo/runtime/session.py

```diff
@@ -491,9 +488,7 @@ def browse_ref(self, external_id):
                 "browse_ref requires a fully qualified parameter: "
                 "'module.identifier'"
             )
-        ir_model_data = self.registry('ir.model.data')
-        module, name = external_id.split('.', 1)
-        return ir_model_data.get_object(self.cr, self.uid, module, name)
+        return self.env.ref(external_id).browse()
```

Comment:
Not sure .browse() is necessary? I think self.env.ref() already return a browse record, isn't it?